src/main/java/com/puppycrawl/tools/checkstyle/DetailNodeTreeStringPrinter.java

```diff
@@ -172,34 +168,4 @@ private static DetailNode parseFile(File file) throws IOException {
             System.getProperty("file.encoding", "UTF-8"));
         return parseJavadocAsDetailNode(text.getFullText().toString());
     }
-
-    /**
-     * Creates DetailAST block comment to pass it to the Javadoc parser.
-     * @param content comment content.
-     * @return DetailAST block comment
-     */
-    private static DetailAST createFakeBlockComment(String content) {
```

Comment:
relocated method to `JavadocUtils` and made it public, it was needed for UTs added. Would that be bad design ? This method, though currently utilized only by class `DetailNodeTreeStringPrinter` has a general nature to it as its name suggests and it might be needed for other things too in future whenever some needs to parse javadoc using [`JavadocDetailNodeParser#parseJavadocAsDetailNode`](https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/JavadocDetailNodeParser.java#L100) since it only accepts argument of type `DetailAST`. Another approach would be to rewrite this logic again in my UT.